src/features/views/components/ViewDataTable/index.tsx

```diff
@@ -93,15 +93,16 @@ const ViewDataTable: FunctionComponent<ViewDataTableProps> = ({
   const [, accessLevel] = useAccessLevel();
 
   const [quickSearch, setQuickSearch] = useState('');
-  const router = useRouter();
-  const { orgId } = router.query;
+  const { orgId } = useNumericRouteParams();
+
   const { showSnackbar } = useContext(ZUISnackbarContext);
   const { showConfirmDialog } = useContext(ZUIConfirmDialogContext);
-
-  const model = useViewDataModel();
-  const browserModel = useModel(
-    (env) => new ViewBrowserModel(env, parseInt(orgId as string))
+  const { addColumn, addPerson, deleteColumn } = UseViewDataTableMutations(
+    orgId,
+    view.id
   );
+  const createView = useCreateView(orgId);
+  const viewGridHook = useViewGrid(orgId, view.id);
```

Comment:
The _hook_ is the function. Just call this variable `viewGrid`, `viewGridFunctions` or `viewGridMutations`, or something like that.